build-logic/jvm/src/main/kotlin/gradlebuild.unittest-and-compile.gradle.kts

```diff
@@ -322,6 +323,10 @@ fun configureTests() {
             val supportedTask = taskIdentity.taskType == Test::class.java
 
             predictiveSelection {
+                this as PredictiveTestSelectionExtensionInternal
+                // GitHub actions for contributor PRs uses public build scan instance
+                // in this case we need to explicitly configure the PTS server
+                server.set(uri("https://ge.gradle.org"))
```

Comment:
@blindpirate Does this work even when the user is not authenticated with `ge.gradle.org`? I suppose it doesn't.

Reply:
I tested in my personal account, which has no token configured at all: https://github.com/blindpirate/gradle/actions/runs/2307268014

Reply:
@wolfs It does as long as ge.gradle.org allows anonymous viewing of scans.